.cascade-code/Chapman.edu/_cascade/formats/level/Masthead.vtl

```diff
@@ -3,322 +3,178 @@
+	#set ( $rotateSpeed = $_XPathTool.selectSingleNode($masthead, 'boxes/speed').value )
+	#set ( $slides = $_XPathTool.selectNodes($masthead, 'boxes/slides/slide') )
+
+	## In the future, let's not use the DOM Tree as a datastore like this. Data attributes
+	## would be better practice.
+	<div class="hidden slideOptions">
+		<div class="autoRotate">
+			${_EscapeTool.xml($autoRotate)}
+		</div>
+
+		<div class="startingSlideNumber">
+			${_EscapeTool.xml($slideStartNumber)}
+		</div>
+
+		<div class="speed">
+			${_EscapeTool.xml($rotateSpeed)}
+		</div>
+	</div>
+
+	<div class="miniRotatorContainer">
+		<ul class="miniRotator clearfix">
+			#foreach($slide in $slides)
+				#set ( $slideInternalLinkPath = $_XPathTool.selectSingleNode($slide, 'internalLink/path').value )
+				#set ( $slideLinkText = $_XPathTool.selectSingleNode($slide, 'link').value )
+				#set ( $slideLinkTarget = $_XPathTool.selectSingleNode($slide, 'linkTarget').value )
+				#set ( $slideImagePath = $_XPathTool.selectSingleNode($slide, 'image/path').value )
+				#set ( $slideImageAltText = $_XPathTool.selectSingleNode($slide, 'altText').value )
+				#set ( $slideHeader = $_XPathTool.selectSingleNode($slide, 'header').value )
+				#set ( $slideDescription = $_XPathTool.selectSingleNode($slide, 'description').value )
+				#set ( $hasMore = ($slideInternalLinkPath != '/' || $slideLinkText != '') && $slideImagePath != '/' )
+
+				## Compute link attributes.
+				#set ( $slideItemHref = '#' )
+				#set ( $slideItemTarget = '' )
+
+				#if ( $slideInternalLinkPath != '/' )
+					#set ( $slideItemHref = $slideInternalLinkPath )
+				#elseif ( $slideLinkText != '' && $slideLinkTarget == 'New Window' )
+					#set ( $slideItemHref = $slideLinkText )
+					#set ( $slideItemTarget = '_blank' )
+				#elseif ( $slideLinkText != '' )
+					#set ( $slideItemHref = $slideLinkText )
+				#else
+					## Use defaults above.
+				#end
+
+				<li>
+					<a class="item" href="${_EscapeTool.xml($slideItemHref)}">
+						<img height="220"
+							 width="266"
+							 alt="${_EscapeTool.xml($slideImageAltText)}"
+							 src="[system-asset]${slideImagePath}[/system-asset]"/>
+
+						<div class="caption ellipsis multiline">
+							<div class="inner">
+								<div class="title">${_EscapeTool.xml($slideHeader)}</div>
+								<div class="copy ellipsis multiline">
+									${_EscapeTool.xml($slideDescription)}
+								</div>
+								#if ( $hasMore )
+									<div class="button">More <span class="bullet">»</span></div>
+								#end
+							</div>
+						</div>
+					</a>
+				</li>
+			#end
+		</ul>
+	</div>
+	<div class="miniRotatorNav">
+		<a class="prev" title="Move back to the previous slide"></a>
+		<a class="next" title="Move forward to the next slide"></a>
+	</div>
 
 #else
-    <!-- mastheadType: base case (no match, no masthead) -->
-
+	<!-- mastheadType: base case (no match, no masthead) -->
```

Comment:
The spacing is off on this file it looks like it was 2 spaces and is now 4 spaces can you change this

Reply:
Updated. My Text Editor must have converted two spaces to four. Let me know if this file still isn't displaying correctly for you.